crates/aptos-admin-service/src/server/consensus/mod.rs

```diff
@@ -0,0 +1,239 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::server::utils::{reply_with, reply_with_status, spawn_blocking};
+use anyhow::{bail, Error};
+use aptos_consensus::{
+    persistent_liveness_storage::PersistentLivenessStorage,
+    quorum_store::{quorum_store_db::QuorumStoreStorage, types::PersistedValue},
+};
+use aptos_consensus_types::{block::Block, common::Payload};
+use aptos_crypto::HashValue;
+use aptos_logger::info;
+use aptos_types::transaction::SignedTransaction;
+use http::header::{HeaderValue, CONTENT_LENGTH};
+use hyper::{Body, Request, Response, StatusCode};
+use std::{collections::HashMap, sync::Arc};
+
+pub async fn handle_dump_consensus_db_request(
```

Comment:
This looks very barebones - pretty much dumping the DB (literally). 

Do we have plans to add user-friendly APIs to inspect the consensus DB like we do with Aptos DB via the Aptos API?

Reply:
Any specific API in your mind? I'm under an impression that the consensus db won't be too large (assuming quorum store is enabled), so I didn't add those query by epoch/round/block_id api.